x-pack/plugins/osquery/cypress/integration/superuser/add_integration.spec.ts

```diff
@@ -10,20 +10,20 @@ import { addIntegration } from '../../tasks/integrations';
 
 import { login } from '../../tasks/login';
 // import { findAndClickButton, findFormFieldByRowsLabelAndType } from '../../tasks/live_query';
-import { ArchiverMethod, runKbnArchiverScript } from '../../tasks/archiver';
+// import { ArchiverMethod, runKbnArchiverScript } from '../../tasks/archiver';
```

Comment:
Why are these commented out?

Reply:
These tests had dependency on agent 8.1, I am not sure if this is ready yet. But it's a good oportunity to double check ;) Thanks!

Reply:
Yeah, I checked. Apparently Fleet has been changed, so these tests have to be adjusted before bringing them back.